webpack/scenes/Content/ContentPage.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import useDeepCompareEffect from 'use-deep-compare-effect';
+import { Grid, GridItem, TextContent, Text, TextVariants } from '@patternfly/react-core';
+import { useSelector, useDispatch } from 'react-redux';
+import { STATUS } from 'foremanReact/constants';
+import { translate as __ } from 'foremanReact/common/I18n';
+import ContentTable from './Table/ContentTable';
+import { getContentTypes } from './ContentActions';
+import { selectContentTypes, selectContentTypesStatus } from './ContentSelectors';
+import { contentConfig, defaultConfig, getConfiguredTypes, parseEnabledTypesResponse } from './ContentConfig';
+import Loading from '../../components/Loading';
+
+const ContentPage = () => {
+  const dispatch = useDispatch();
+  const enabledTypesResponse = useSelector(selectContentTypes);
+  const enabledTypesStatus = useSelector(selectContentTypesStatus);
+  const [contentTypeSelected, setContentTypeSelected] = useState(null);
+  const [typeSelectedConfig, setTypeSelectedConfig] = useState(null);
```

Comment:
nitpick: Thoughts on renaming these to selectedContentType and selectedContentTypeConfig?

Reply:
I agree :)

Feel free to comment on the names of other things too. Sometimes naming is hard.